benchmarks/torchbench_model.py

```diff
@@ -64,59 +65,51 @@
     "cm3leon_generate": [
         {
             "test": "train",
-        },
+        },  # Model's DEFAULT_TRAIN_BSIZE is not implemented
         {
             "test": "eval",
             "xla": "PJRT",
-        },
-    ],  # no install.py
+        },  # TIMEOUT
+    ],
     "hf_T5_generate": [
         {
             "test": "train",
-        },
+        },  # Model's DEFAULT_TRAIN_BSIZE is not implemented
         {
             "test": "eval",
             "xla": "PJRT",
-        },
-    ],  # no install.py
+        },  # TIMEOUT
+    ],
     "doctr_det_predictor": [{
         "test": "train"
-    },],  # not implemented
+    },],  # Model's DEFAULT_TRAIN_BSIZE is not implemented
     "doctr_reco_predictor": [{
         "test": "train"
-    },],  # not implemented
+    },],  # Model's DEFAULT_TRAIN_BSIZE is not implemented
     "detectron2_fcos_r_50_fpn": [{
         "test": "train"
-    },],  # not implemented
-    # https://github.com/pytorch/torchdynamo/issues/145
-    "fambench_xlmr": [{}],
-    "llama": [{
-        "test": "train"
-    },],  # not implemented
+    },],  # FCOS train is not supported by upstream detectron2
     "mobilenet_v2_quantized_qat": [
         {
             "test": "eval",
             "accelerator": "cuda"
-        },  # not implemented
+        },  # The eval test only supports CPU
         {
             "test": "eval",
             "accelerator": "tpu"
-        },
-    ],  # not implemented
+        },  # The eval test only supports CPU
+    ],
     # self.load_benchmark() exits the main process. See issue #6207.
     "pytorch_CycleGAN_and_pix2pix": [{}],
     "pyhpc_equation_of_state": [{
         "test": "train"
-    },],  # not implemented
+    },],  # Model's DEFAULT_TRAIN_BSIZE is not implemented
     "pyhpc_isoneutral_mixing": [{
         "test": "train"
-    },],  # not implemented
+    },],  # Model's DEFAULT_TRAIN_BSIZE is not implemented
     "pyhpc_turbulent_kinetic_energy": [{
         "test": "train"
-    },],  # not implemented
-    "pytorch_struct": [{
-        "test": "eval"
-    },],  # not implemented
+    },],  # Model's DEFAULT_TRAIN_BSIZE is not implemented
```

Comment:
Thanks for the refining. Does those models removed from the list work fine now?

Reply:
Not really. But, as far as I can see, they are not in torchbench list of models anymore